config/config.php

```diff
@@ -36,11 +36,22 @@
 /**
  * Hooks
  */
-$GLOBALS['TL_HOOKS']['parseArticles'][] = array('News', 'addCategoriesToTemplate');
-$GLOBALS['TL_HOOKS']['replaceInsertTags'][] = array('News', 'parseCategoriesTags');
+$GLOBALS['TL_HOOKS']['parseArticles']['news_categories']     = ['News', 'addCategoriesToTemplate'];
+$GLOBALS['TL_HOOKS']['replaceInsertTags']['news_categories'] = ['News', 'parseCategoriesTags'];
+$GLOBALS['TL_HOOKS']['replaceInsertTags']['news_categories'] = ['InsertTags', 'replace'];
 
-if (in_array('changelanguage', \ModuleLoader::getActive())) {
-    $GLOBALS['TL_HOOKS']['translateUrlParameters'][] = array('NewsCategories', 'translateUrlParameters');
+
+if (in_array('changelanguage', \ModuleLoader::getActive()))
+{
+    $GLOBALS['TL_HOOKS']['translateUrlParameters'][] = ['NewsCategories', 'translateUrlParameters'];
+}
+
+foreach ($GLOBALS['TL_HOOKS']['getSearchablePages'] as &$arrCallback)
+{
+    if ($arrCallback[0] == 'News' && $arrCallback[1] == 'getSearchablePages')
+    {
+        $arrCallback = ['News', 'getSearchablePages'];
```

Comment:
We need to adjust the news urls, for that reason, we simply change the contao core \Contao\News::getSearchablePages to news_categories News::getSearchablePages. \Contao\News::getSearchablePages creates the news link through the News::getLink method, which we overwrite in news_categories News:.getLink that inheritance from \Contao\News.